openshift/README.md

```diff
@@ -0,0 +1,75 @@
+# Build and deploy on OpenShift Container Platform
+
+The loopback4-example-shopping/openshift contains Dockerfiles for building
+container images suitable for running on OpenShift Container Platform and yaml
+files for creating build configurations for building the images. They have been
+tested on OCP 4.6.9.
+
+## First build and deploy
+
+First run these commands in loopback4-example-shopping/openshift to create a new
```

Comment:
do they have to be run in order? if yes maybe mention the same

Reply:
I think most people will try to run them in the order they are written and that will work.
I think it will add very little value explaining what commands can be run in a different order than listed.

Reply:
I think we missed each other the recommendation was to mention that the comments should be executed in order. Remember not everyone who tries this out will be experienced in OpenShift. Change the line to:

```
First run the following commands in order in loopback4-example-shopping/openshift to create a new
```